internal/schemautil/userconfig/apiconvert/toapi.go

```diff
@@ -269,12 +267,7 @@ func itemToAPI(
 
 		va, ok := v.([]interface{})
 		if !ok {
-			// This can be TypeSet
-			set, ok := v.(*schema.Set)
```

Comment:
`apiconvert` is not used when converting the `user_peer_network_cidrs` field, i.e this is not needed